test/unit/org/apache/cassandra/Util.java

```diff
@@ -783,6 +784,13 @@ public static UnfilteredPartitionIterator executeLocally(PartitionRangeReadComma
         return command.queryStorage(cfs, controller);
     }
 
+    public static UnfilteredPartitionIterator executeLocally(SinglePartitionReadCommand command,
```

Comment:
nit: Can we just make `ReadCommand#queryStorage()` public and merge these two methods?

Reply:
atm we only have 2 types, but wonder if we need to worry about adding methods to `ReadCommand` as other indexes would be aware?  I "feel" like adding is safe...

Reply:
thought about this more and I rather not change virtual tables in this patch.  I asked this question in slack (see https://the-asf.slack.com/archives/CK23JSY2K/p1678145643633279) and it may make sense to make public and depend on it, but would need to flesh out virtual tables implementation as they don't use that method and will have failures if touched.